src/plugins/unified_field_list/common/utils/field_examples_calculator.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+// Adapted from src/plugins/discover/public/application/main/components/sidebar/lib/field_calculator.js
+
+import { map, sortBy, defaults, isObject, pick } from 'lodash';
+import * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import type { DataView, DataViewField } from '@kbn/data-plugin/common';
+import { flattenHit } from '@kbn/data-plugin/common';
+
+type FieldHitValue = any;
+
+interface FieldValueCountsParams {
+  hits: estypes.SearchHit[];
+  dataView: DataView;
+  field: DataViewField;
+  count?: number;
+}
+
+export const canFetchFieldExamples = (field: DataViewField): boolean => {
+  return !(
+    field.type === 'geo_point' ||
+    field.type === 'geo_shape' ||
+    field.type === 'attachment' ||
+    field.type === 'unknown'
```

Comment:
we should add `_score`
![Bildschirmfoto 2022-09-09 um 14 32 06](https://user-images.githubusercontent.com/463851/189352588-7608e6b1-84d8-491f-8ee8-395b02f39709.png)
```suggestion
    field.type === 'unknown' ||
    field.name === '_score'
```
I wonder if we shouldn't switch to logic to check for `field.type`s we're sure are working. For the case, that an `unknown` field is implemented, so it has a type e.g.`thanos`. Given it can't fetch field examples or fails 50%  the cases, it wouldn't trigger the error like `_score`

Reply:
@kertal Inverted the logic to check for known types